### PR TITLE
Feat/update mentoring gsoc

### PR DESCRIPTION
**Pull request by Honyii**
**Issues**:
 - Closes [#847 ](https://github.com/json-schema-org/community/issues/847)

### CODE REVIEW

**Comment by Honyii** [comment]
closes #847

File: programs/mentoring/gsoc/README.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ We are a community JSON Schema enthusiast dedicated to maintain, evolve and prom
...
-### Current GSoC edition (2024) project ideas
+### Current GSoC edition (2025) project ideas
 
-You can find the project ideas for this year's program [here](gsoc-2024.md).
+Project ideas for this year's program are forthcoming. Check back soon [here](gsoc-2025.md).
 
 ### Past GSoC editions project ideas
 
 - 2023 [ideas](gsoc-2023.md)
+- 2023 [ideas](gsoc-2024.md)
```
Comment (inclinedadarsh) [inline programs/mentoring/gsoc/README.md:21]: This can be updated to `2024`

Comment (Honyii) [reply programs/mentoring/gsoc/README.md]: edited @inclinedadarsh

**Review by benjagm — APPROVED** [review]
Great work!

**Pull request merged**